test/src/us/kbase/kbasefeaturevalues/test/KBaseFeatureValuesServerTest.java

```diff
@@ -126,7 +126,7 @@ public static void init() throws Exception {
         File inputFile = new File(inputDir, "Desulfovibrio_vulgaris_Hildenborough_microarray_log_level_data.txt");
         Map<String, Object> contigsetData = new LinkedHashMap<String, Object>();
         contigsetData.put("contigs", new ArrayList<Object>());
-        contigsetData.put("id", "1945.contigset");
+        contigsetData.put("id", "1945.assembly");
```

Comment:
Adding an assembly test is good, but the contig set test should be left in place to test for regressions on handling contig sets, which still exist in the system.

Reply:
Well, the Assembly and ContigSet components are removed because they complicated the tests and none of that data is actually ever used in these apps. These apps just need the Genome object to get a list of feature ids. So the tests run on somewhat slimified Genome objects though slimming only applies to handles and Assembly/ContigSet.

Reply:
If the app never touches the dependency objects, is it possible to save a genome without either the assembly or contigset? If so do it that way. Having the CS or Assembly as part of the tests will make people think they're actually tested and important.